testcases/PowerNVDump.py

```diff
@@ -804,7 +805,7 @@ def runTest(self):
         self.setup_pwdless_auth()
         self.setup_test("net")
         self.setup_ssh()
-        log.debug("=============== Testing kdump/fadump over ssh ===============")
+        log.info("=============== Testing kdump/fadump over ssh ===============")
```

Comment:
push these change in another commit please

Reply:
Raised new PR https://github.com/open-power/op-test/pull/713